tests/Microsoft.Fx.Portability.Tests/Microsoft.Fx.Portability.Tests.csproj

```diff
@@ -24,7 +21,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;FEATURE_XML_SCHEMA</DefineConstants>
```

Comment:
Why is FEATURE_XML_SCHEMA defined? I don't see any changes here that require it.